goblint.opam.template

```diff
@@ -1,3 +0,0 @@
-pin-depends: [
```

Comment:
This probably shouldn't be removed. We want the pinned version since we don't release to opam regularly.